docs/userguide/failover/failover-overview.md

```diff
@@ -15,26 +15,133 @@ The following describes some scenarios of multi-cluster failover:
+
+`PurgeMode` represents how to deal with the legacy applications on the cluster from which the application is migrated. Currently, the supported values are `Directly` and `Gracefully`, with the default being `Gracefully`. The explanations for each type are as follows:
+
+- `Directly` represents that Karmada will directly evict the legacy application. This is useful in scenarios where an application cannot tolerate two instances running simultaneously. For example, the Flink application supports exactly-once state consistency, which means it requires that no two instances of the application are running at the same time. During a failover, it is crucial to ensure that the old application is removed before creating a new one to avoid duplicate processing and maintaining state consistency.
+- `Gracefully` represents that Karmada will wait for the application to become healthy on the new cluster or after a timeout is reached before evicting the application.
+
+PropagationPolicy can be configured as follows:
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-propagation
+spec:
+  failover:
+    cluster:
+      purgeMode: Directly
+  #...
 ```
---feature-gates=Failover=true,GracefulEviction=true
+
+### Application State Preservation
+
+Starting from version v1.15, the cluster failover feature has added support for stateful applications, providing users with a general way to define application state preservation in the context of cluster failover.
+
+#### StatePreservation Introduction
+
+Karmada currently supports propagating various types of resources, including Kubernetes objects and CRDs (Custom Resource Definitions), which also covers stateful workloads. This enables distributed applications to achieve multi-cluster resilience and elastic scaling. Stateless workloads have the advantage of higher fault tolerance, as the loss of a workload does not affect the correctness of the application and does not require data recovery. In contrast, stateful workloads (such as Flink) rely on checkpoints or saved state to recover after a failure. If this state is lost, the job may not be able to resume from where it left off.
+
+For workloads, failover scenarios include two cases: the first is when the workload itself fails and needs to be migrated to another cluster; the second is when the cluster experiences a failure, such as node crashes, network partitions, or control plane outages, requiring the migration of some workloads within the cluster.
+
+[Karmada completed the design and development of the first scenario in version v1.12](application-failover.md#application-state-preservation). Starting from v1.15, we support the second scenario—cluster failover for stateful application. When the migration mode is set to `Directly`, the Karmada system will first remove the application from the original cluster. Only after the application has been completely removed can it be scheduled to the new cluster.
```

Comment:
It may seem unnecessary to explain the history here. All we need to do is explain what state preservation is.